crates/gossip/src/state.rs

```diff
@@ -24,7 +24,10 @@ pub(crate) struct GossipRoundState {
     ///
     /// Note that it's actually [kitsune2_api::OpId]s that are exchanged. So this is a hint in
     /// terms of op data about how many op ids we should send back to the other peer.
-    pub peer_max_op_data_bytes: u32,
+    ///
+    /// The value is an `i32` because it's a soft limit and in order to send complete slices,
+    /// gossip may exceed the limit. That is recorded here as a negative number.
```

Comment:
What would a value like -100 indicate? The number of bytes can be the next complete slice greater than 100 bytes?